src/pages/LoginPage.js

```diff
@@ -33,35 +36,44 @@ function Login() {
   }, [navigate]);
 
   return (
-    <div className="LoginPage">
-      <div className="container">
+    <div className="LoginPage d-flex  justify-content-center align-items-center">
+      <Container className=" d-flex justify-content-between bg-white rounded-4">
         <div className="right">
```

Comment:
This `div` can be `main` or `section` and className should be changed with something like `login-form-section`.

Reply:
Şu anda `<div className="login-form-section">` olan kısmı `main` yapmamız gerekiyordu, şu anda `main` olan kısmın `div` olarak kalması gerekiyordu.